onboarding-manager/internal/onboarding/types/types.go

```diff
@@ -52,6 +52,8 @@ type (
 		SSHKey string
 		// OS resource ID
 		OSResourceID string
+		// CustomConfigs is a map of custom cloud init (e.g. name:cloud-init-config)
```

Comment:
we should say to use name => cloud init as the name is unique and immutable. Shall we include tenantID ?

Reply:
Uniqueness of name in the in-memory map should suffice as the key is anyways discarded, only the configs flow to the Edge Node.